src/pkg/logs/logrus.go

```diff
@@ -17,3 +23,44 @@ func IsLogrusError(message string) bool {
 		return true // show by default (likely Dockerfile errors)
 	}
 }
+
+type TermLogFormatter struct {
+	Term *term.Term
+}
+
+func (f TermLogFormatter) Format(entry *logrus.Entry) ([]byte, error) {
+	getMessage := func() string {
+		var buf strings.Builder
+		buf.WriteString(entry.Message)
+		for k, v := range entry.Data {
+			buf.WriteString(" ")
+			buf.WriteString(k)
+			buf.WriteString("=")
+			buf.WriteString(v.(string))
```

Comment:
is this safe?

Reply:
Good catch, changed to use fmt.Fprintf with %v